webaverse/app
# Walk/run animation blending

Issue related to this PR:
https://github.com/webaverse/app/issues/1695

Steps for testing:
Walk around
Press Shift to start running
Switch from walk to run and vice versa should not be instant
